.goreleaser.yaml

```diff
@@ -17,7 +17,7 @@ archives:
 checksum:
   name_template: 'checksums.txt'
 snapshot:
-  name_template: "{{ incpatch .Version }}-next"
+  name_template: "{{ incpatch .Version }}-0next"
```

Comment:
I'm kinda curious how we fit this into the release process. 

We cut a tag which ultimately runs the [release action](https://github.com/retr0h/gilt/blob/main/.github/workflows/release.yml#L28), which of course uses the current tag for versioning.

I'm thinking we want to do something similar with `task build:wheel`, where we build wheels which follow the same versioning.

Seems like we would want the wheel to be named `gilt-2.0.4-0-py3-none-macosx_10_15_x86_64.whl` vs `gilt-2.0.4-0next-py3-none-macosx_10_15_x86_64.whl`, right?

Reply:
Looks like we can leverage `GORELEASER_CURRENT_TAG=$(git describe --tags --abbrev=0)` (or similar) to force `--snapshot` to use the current tag when building.

Reply:
It's supposed to use whatever the `version:`field in the `metadata.json` for the current build.  I was under the impression that the release action would use the current tag for that value, and it would be, e.g., just `2.0.4`.

TL;DR: maybe it will Just Work™?

Reply:
Oh, I see the issue you're describing.  `task` is not `make`, and `task build:wheel` will unconditionally rebuild the binary every time, which would get you a "wrong" version number if you ran it post-release, because of how `task build` invokes Goreleaser.

So you'd want the release action to instead run `python/dist2wheel.py` directly, so it can use the `dist/` directory with the released artifacts in it.

There's a simple fix for this.  One sec.